common/src/leap/soledad/common/couch.py

```diff
@@ -476,9 +434,14 @@ def __init__(self, url, dbname, replica_uid=None, ensure_ddocs=True):
             self._set_replica_uid(replica_uid)
         if ensure_ddocs:
             self.ensure_ddocs_on_db()
-        # initialize a thread pool for parallelizing get_docs()
-        self._sem_pool = threading.BoundedSemaphore(
-            value=self.MAX_GET_DOCS_THREADS)
+        self._cache = None
```

Comment:
> Why does the couchdb code live in soledad.common anyway?
@andrejb would know better, but I think that at some point it was intended to avoid the mx->soledad dep. (it might have been early in the development). Right now, it might be good to move it.